Analysis/Tasks/PWGHF/qaTask.cxx

```diff
@@ -55,209 +32,130 @@ double ConvertPhiRange(double phi)
 /// \param impactParameterZ variable to save the impact parameter (in Z) in micrometers.
 /// \param impactParameterZError variable to save the impact parameter (in Z) error in micrometers.
 template <typename Track>
```

Comment:
Add documentation of the return value with `/// \return true if...`

Reply:
I would rather open another PR instead of adding commits to this one, I can remove the function if you don't agree